Package.swift

```diff
@@ -1,5 +1,20 @@
+// swift-tools-version:4.0
+
 import PackageDescription
 
 let package = Package(
-    name: "LambdaKit"
+    name: "LambdaKit",
+    products: [
+        .library(
+            name: "LambdaKit",
+            targets: ["LambdaKit"]),
+    ],
+    dependencies: [
```

Comment:
you should be able to omit this one

Reply:
Actually on all of them https://github.com/apple/swift-package-manager/blob/c62bd0bb8c7669fd284e6e3b2fc82f9708334b4f/Sources/PackageDescription/Package.swift#L86

Reply:
done

Reply:
we can't omit because the file structure is not following the conventional layout. See the "path" parameter.

Reply:
Is there a new standard layout? Since this was the standard before I guess?

Reply:
IMO we should make it whatever the standard is if we can reduce all this nonsense

Reply:
lemme see what can be done.

Reply:
We shouldn't let this block our Swift 4 merge tho, so this is fine if this all works, but we should follow up

Reply:
yeah not going to make this a hard dependency.